src/libs/domain/triggerWelcomeBulle.ts

```diff
@@ -0,0 +1,6 @@
+import { history } from '../../app/profiles/store';
+
+const triggerWelcomeBulle = (): void =>
```

Comment:
Domain should not be dependent of the store... or dependence should at least be explicit :
```
const triggerWelcomeBulle = (history: History): void => history.push('?pk_campaign=installed-contributors-list');
```
Anyway, I don't think this -- navigation -- behavior is part of the domain.

Reply:
ok